pages/assessment.md

```diff
@@ -0,0 +1,17 @@
+# Assessment  
```

Comment:
Need to replace this line with

~~~
---
layout: page-fullwidth
permalink: /assessment/
title: Assessment
redirect_from:
  - /pages/assessment.html
---
~~~